packages/pwa-kit-build/src/configs/webpack/config.js

```diff
@@ -244,7 +253,8 @@ const clientOptional = baseConfig('web')
                 ...optional('worker', './worker/main.js'),
                 ...optional('core-polyfill', resolve(projectDir, 'node_modules', 'core-js')),
                 ...optional('fetch-polyfill', resolve(projectDir, 'node_modules', 'whatwg-fetch'))
-            }
+            },
+            plugins: [analyzeBundle && getBundleAnalyzerPlugin('client-optional')]
```

Comment:
This is nice and short, but it might result in a plugin array that looks like this: 

```
[false]
```

You'll want to filter on `Boolean` to get rid of the false values. You can use [this](https://github.com/webpack/webpack/issues/5493) link for reference